dotCMS/src/main/java/com/dotcms/rest/api/v1/apps/view/SiteView.java

```diff
@@ -42,6 +46,7 @@ public SiteView(final String id,final String name,
         this.name = name;
         this.configured = secrets.stream().anyMatch(secretView -> null != secretView.getSecret());
         this.secrets = secrets;
+        this.secretsWithWarnings = null;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Assigning an Object to null is a code smell.  Consider refactoring.](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5519512)